.drone.yml

```diff
@@ -34,9 +40,27 @@ steps:
     event:
     - push
 
+- name: build-with-tag
+  pull: default
+  image: rancher/dapper:1.11.2
+  commands:
+  - VERSION="${DRONE_TAG}" dapper ci
```

Comment:
I think `build` should pick up the version from git repo automatically?

Reply:
It will pickup git commit not the tag.

Reply:
that's weird. e.g. if you try it locally, it should pick up the tag. see `scripts/version`

Reply:
fixed